tgui/packages/tgui/interfaces/MODsuit.js

```diff
@@ -140,79 +141,88 @@ const HealthAnalyzer = (props, context) => {
-          </Section>
-        </Stack.Item>
-        <Stack.Item grow>
-          <Section title="Suffocation">
+    <Section>
+      {(!!show_vitals && (
+        <>
+          <Section title="Health">
             <ProgressBar
-              value={active ? useroxy / usermaxhealth : 0}
+              value={active ? userhealth / usermaxhealth : 0}
               ranges={{
-                good: [-Infinity, 0.2],
+                good: [0.5, Infinity],
                 average: [0.2, 0.5],
-                bad: [0.5, Infinity],
+                bad: [-Infinity, 0.2],
               }}>
-              <AnimatedNumber value={active ? useroxy : 0} />
+              <AnimatedNumber value={active ? userhealth : 0} />
             </ProgressBar>
           </Section>
-        </Stack.Item>
-      </Stack>
-    </>
+          <Stack textAlign="center">
+            <Stack.Item grow>
+              <Section title="Brute">
+                <ProgressBar
+                  value={active ? userbrute / usermaxhealth : 0}
+                  ranges={{
+                    good: [-Infinity, 0.2],
+                    average: [0.2, 0.5],
+                    bad: [0.5, Infinity],
+                  }}>
+                  <AnimatedNumber value={active ? userbrute : 0} />
+                </ProgressBar>
+              </Section>
+            </Stack.Item>
+            <Stack.Item grow>
+              <Section title="Burn">
+                <ProgressBar
+                  value={active ? userburn / usermaxhealth : 0}
+                  ranges={{
+                    good: [-Infinity, 0.2],
+                    average: [0.2, 0.5],
+                    bad: [0.5, Infinity],
+                  }}>
+                  <AnimatedNumber value={active ? userburn : 0} />
+                </ProgressBar>
+              </Section>
+            </Stack.Item>
+            <Stack.Item grow>
+              <Section title="Toxin">
+                <ProgressBar
+                  value={active ? usertoxin / usermaxhealth : 0}
+                  ranges={{
+                    good: [-Infinity, 0.2],
+                    average: [0.2, 0.5],
+                    bad: [0.5, Infinity],
+                  }}>
+                  <AnimatedNumber value={active ? usertoxin : 0} />
+                </ProgressBar>
+              </Section>
+            </Stack.Item>
+            <Stack.Item grow>
+              <Section title="Suffocation">
+                <ProgressBar
+                  value={active ? useroxy / usermaxhealth : 0}
+                  ranges={{
+                    good: [-Infinity, 0.2],
+                    average: [0.2, 0.5],
+                    bad: [0.5, Infinity],
+                  }}>
+                  <AnimatedNumber value={active ? useroxy : 0} />
+                </ProgressBar>
+              </Section>
+            </Stack.Item>
+          </Stack>
+        </>
+      )) || (
+        <Section>
+          {'Health Analyzer Vitals Readout Disabled In Settings'}
```

Comment:
does this display? i'm not sure why this is enclosed in curly brackets

Reply:
this actually does display. i assume i bracketed it up out of reflex or something but it's visible in one of the test images